<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServisGetDialog</class>
 <widget class="QDialog" name="ServisGetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1265</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Teknik Servis</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QGroupBox" name="DonanimBox_2">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Cihazla gelen donanımlar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="donanim_1">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Çanta</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="donanim_2">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Cihaz</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="donanim_3">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>1. batarya</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="donanim_4">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>2. batarya</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="donanim_5">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>SDI kablo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="donanim_6">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>HDMI kablo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="donanim_7">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Desktop Şarj</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="donanim_8">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Adaptör</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="donanim_9">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>SIM kartlar(1-6)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="donanim_10">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>SIM kartlar slotları (1-6)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="donanim_11">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>CVBS kablo(Görüntü - Ses)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="QGroupBox" name="MusteriAdiBox">
             <property name="styleSheet">
              <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_16">
              <item>
               <widget class="QLabel" name="musteriAdi_2">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Müşteri Adı</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="musteriAdi_1">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QGroupBox" name="SevkBox">
             <property name="styleSheet">
              <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="sevkTarihi_2">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Sevk Tarihi</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="sevkTarihi_1">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QGroupBox" name="ServisNoBox">
             <property name="styleSheet">
              <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_14">
              <item>
               <widget class="QLabel" name="servisNo_2">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Cihaz Seri No</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="servisNo_1">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QGroupBox" name="GelisBox">
             <property name="styleSheet">
              <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_15">
              <item>
               <widget class="QLabel" name="gelisTarihi_2">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Geliş Tarihi</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="gelisTarihi_1">
                <property name="minimumSize">
                 <size>
                  <width>205</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QGroupBox" name="BitisTarihBox">
             <property name="styleSheet">
              <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_18">
              <item>
               <widget class="QLabel" name="bitisTarihi_2">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Tamir Bitiş Tarihi</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="bitisTarihi1">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QGroupBox" name="ArizaTarifBox">
             <property name="styleSheet">
              <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <widget class="QLabel" name="arizaTanimi_2">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Arıza Tarifi</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="arizaTanimi_1">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QGroupBox" name="TestSureBox">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_17">
            <item>
             <widget class="QLabel" name="testSuresi_2">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Test Süresi</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="testSuresi_1">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="YapilanIslemBox">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <widget class="QLabel" name="yapilanIslem_2">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Yapılan İşlem</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="yapilanIslem_1">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="notlar">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Notlar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="notlar_">
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>26</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Güncelle</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
